README.md

```diff
@@ -204,6 +205,7 @@ it will automatically trigger the backfilling process for the allowed repositori
 6. All the builds and deployments data will be backfilled that contain the issue keys. You can check how to include 
    issue keys to the builds and deployments [here](#see-github-builds-and-deployments-in-jira).
 
+If an error occurs during the backfilling process, the app will prompt you to retry the backfilling for the failed repositories without having to restart the entire backfill process, except for permission errors. You have to first resolve the permission errors before retrying the backfill process.
```

Comment:
Edit: "...process, except for permission errors. You have to first resolve the permission errors before retrying the backfill process." to "...process. However, this does not account for permission errors. You will need to resolve any permissions errors before retrying the backfill process."